plexpy/libraries.py

```diff
@@ -143,6 +145,44 @@ def has_library_type(section_type):
     result = monitor_db.select_single(query=query, args=args)
     return bool(result)
 
+def get_library_media_stats(section_id=None, refresh=False):
```

Comment:
Save to the database `library_sections` table instead of caching to a file. Add two new columns `total_storage` and `total_duration`. `total_size` is already the `count` column in the database.

You can keep the `total_size` in the returned dict for the API.